### PR TITLE
group generators by sheet and randomize

### PR DESCRIPTION
**Pull request by michaone007**

### CODE REVIEW

**Comment by michaone007** [comment]
need to discuss this.

File: app/src/main/java/com/example/zakiva/santa/Models/Generator.java [hunk 1]
```diff
@@ -65,6 +67,16 @@ public TriviaQuestion geoSeq(){
 
         return new TriviaQuestion("33", "בסדרה זו מה המספר הבא " + sidra + " ?", s, getNumericAnswers(n, 5).get(0), getNumericAnswers(n, 5).get(1), s, getNumericAnswers(n, 5).get(2),false);
     }
+    public TriviaQuestion generateIsraelBandQuestion(){
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/Models/Generator.java]: groupGeneratorsIsraelBands

File: app/src/main/java/com/example/zakiva/santa/Models/Generator.java [hunk 2]
```diff
@@ -65,6 +67,16 @@ public TriviaQuestion geoSeq(){
...
+        ArrayList<TriviaQuestion> array = new ArrayList<>();
+        array.add(israelBandToYear());
+        array.add(israelBandWhoLast());
+        array.add(israelBandWhoFirst());
+        array.add(israelBandToMembers());
+        array.add(israelBandToAlbum());
+        Collections.shuffle(array);
+        return array.get(0);
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/Models/Generator.java]: nice

**Comment by michaone007** [comment]
what do you think? 
should we add another random level or its enough?

**Comment by zakiva** [comment]
WTF lol

**Comment by michaone007** [comment]
ready to merge

File: app/src/main/java/com/example/zakiva/santa/Models/Generator.java [hunk 3]
```diff
@@ -65,6 +68,142 @@ public TriviaQuestion geoSeq(){
...
+        }return tq;
+    }
+    public TriviaQuestion groupGeneratorsLatitudes() {
+        TriviaQuestion tq = null;
+        int random = new Random().nextInt(2);
+        switch (random) {
+            case 0:tq = mostNorth();break;
+            case 1:tq = mostSouth();break;
+        }return tq;
+    }    public TriviaQuestion groupGeneratorsMaleActors() {
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/Models/Generator.java]: missing new line here.
Also, merge this 2 groups to 1 group. 
groupGeneratorsActors()
(no mail / female)

Comment (michaone007) [reply app/src/main/java/com/example/zakiva/santa/Models/Generator.java]: k

**Comment by michaone007** [comment]
READY to merge

**Pull request merged**